components/Card.js

```diff
@@ -6,12 +6,14 @@ import Image from 'next/image'
 export default function Card(props) {
   return (
     <div
-      className={`h-auto w-48 mx-auto md:w-44 block ${
+      className={`h-auto w-48 mx-auto md:w-44 focus:rounded-lg focus:ring focus:ring-gray-600  ${
         props.selected
-          ? `border-4 border-canadaBlue rounded-lg`
-          : `border border-slate-300`
-      }`}
+          ? `border-4 border-canadaBlue rounded-lg focus:ring-0`
+          : `border border-slate-300 `
+      } ${props.className ? props.className : ``}`}
```

Comment:
Do we need this ternary operator? `${props.className ? props.className : ``}`

If no props.className is passed, the defaultProps is set to an empty string.

Reply:
Good catch, will push a small change